src/place/form.jsx

```diff
@@ -0,0 +1,90 @@
+const template = (configContext) => {
+  const {
+    React,
+  } = configContext.lib;
+
+  const {
+    Panel,
+  } = configContext.layoutComponents;
+
+  const {
+    Field,
+  } = configContext.recordComponents;
+
+  return (
+    <Field name="document">
```

Comment:
Similarly, the `<Field name="document">` should not be included in this file, so that consumers of this plugin can include this layout inside their form config where they want it. Use the `claim` and `collectionobject` forms in this project as examples.